ccm_web/client/src/components/APIErrorMessage.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Typography } from '@material-ui/core'
+
+import { CanvasError } from '../utils/handleErrors'
+
+interface APIErrorMessageProps {
+  context: string
+  error: Error | undefined
+}
+
+const extractErrorText = (error: Error): string[] => {
+  if (error instanceof CanvasError) {
+    return error.errors.map(e => e.message)
+  } else {
+    return [error.message]
```

Comment:
This logic seems to be checking for various error format from backend, Canvas. Do you think this needs to be revisited again from the merging of add Non-UM user feature. Since The Backend error Object **Might** changed?

Reply:
I think we can merge this code as is but when it is time to integrate with Non-UM UI with Backend, this component  can be adjusted. I don't think I want to put that as criteria for merging this.

Reply:
As I told you in the meeting, I may not use this component, or only partially use it for the Add Non UM Users integration; we'll see how it shakes out.